src/shared/utils/createRtApiConfig.ts

```diff
@@ -14,6 +14,19 @@ export const createRtApiConfig = (userKey: string) => {
   return config;
 };
 
+export const createRtApiContactConfig = (userKey: string) => {
+  const config = {
+    headers: {
+      Authorization: `Userkeyauth ${base64encode(
+        `${REDTAIL_API_KEY}:${userKey}`
+      )}`,
+      include: "addresses,phones,emails,urls",
```

Comment:
This is the first i've seen this `include` param, can you explain what it's doing here?